### PR TITLE
Bug Fix | Admin Login Loop

### PR DESCRIPTION
**Pull request by ChazUK**

### CODE REVIEW

**Comment by ChazUK** [comment]
[(1) [## 8070 ##] Admin users logging in to ASC-WDS](https://trello.com/c/yCIhGrzi/249-1-8070-admin-users-logging-in-to-asc-wds)

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -73,11 +75,11 @@ const routes: Routes = [
   },
   {
     path: '',
-    canActivate: [HasPermissionsGuard],
+    canActivate: [AuthGuard, HasPermissionsGuard],
```
Comment (omarCreativeDev) [inline src/app/app-routing.module.ts]: does the AuthGuard need to be here? its already on line 77 at a higher level as canActivateChild...

Comment (ChazUK) [reply src/app/app-routing.module.ts]: It's here so that it can cancel the HasPermissionsGuard API call if there is no Auth Token

Comment (ChazUK) [reply src/app/app-routing.module.ts]: removed

**Pull request merged**